lib/clearance/test_unit.rb

```diff
@@ -1,8 +1,8 @@
 require 'clearance/testing/deny_access_matcher'
-require 'clearance/testing/helpers'
+require 'clearance/testing/controller_helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.